addons/web/static/src/views/kanban/kanban_renderer.xml

```diff
@@ -57,13 +57,15 @@
                             </div>
                             <div t-if="(env.isSmall or !group.isFolded) and group.progressBars.length" class="o_kanban_counter position-relative d-flex align-items-center justify-content-between" t-att-class="{ 'opacity-25': !group.count }">
                                 <div class="o_kanban_counter_progress progress w-75 bg-300">
+                                    <t t-set="maxWidth" t-value="100 - Math.max(0, group.progressBars.filter(x => x.count > 0).length - 1) * 5"/>
```

Comment:
As this is more complex than a simple division, maybe progressBar should return a value widht (as it returns color and count).

This can be done in _generateProgressBars() on kanban_model.js